frontend/src/lib/components/Annotations/AnnotationMarker.tsx

```diff
@@ -91,7 +89,8 @@ export function AnnotationMarker({
     const { user } = useValues(userLogic)
     const { currentTeam } = useValues(teamLogic)
     const { currentOrganization } = useValues(organizationLogic)
-    const { diffType, groupedAnnotations } = useValues(annotationsLogic({ insightNumericId }))
+    // insightAnnotationsLogic's props must be bound using BindLogic - this is so that we can avoid prop drilling here
```

Comment:
To avoid this and similar comments, you can throw this message as an error in the logic's key function.